src/Symfony/Component/DependencyInjection/Attribute/Target.php

```diff
@@ -21,11 +22,18 @@
 #[\Attribute(\Attribute::TARGET_PARAMETER)]
 final class Target
 {
-    public string $name;
+    public function __construct(
+        public ?string $name = null,
+    ) {
+    }
 
-    public function __construct(string $name)
+    public function getParsedName(): string
```

Comment:
should we even have it as a public method if calling it directly would be a mistake ?

Reply:
I think yes. Calling it directly work, nut only if you gave a name to the constructor.